sentry_logging/lib/src/extension.dart

```diff
@@ -0,0 +1,54 @@
+// ignore_for_file: public_member_api_docs
+
+import 'package:logging/logging.dart';
+import 'package:sentry/sentry.dart';
+import 'package:sentry/sentry_io.dart';
+
+extension LogRecordX on LogRecord {
+  Breadcrumb toBreadcrumb() {
+    return Breadcrumb(
+      category: 'log',
+      type: 'debug',
+      timestamp: time,
+      level: level.toSentryLevel(),
+      message: message,
+      data: <String, Object>{
+        if (object != null) 'LogRecord.object': object!.toString(),
+        if (error != null) 'LogRecord.error': error!.toString(),
+        if (stackTrace != null) 'LogRecord.stackTrace': stackTrace!.toString(),
```

Comment:
since the fallback of serialization calls `.toString()`, we probably could avoid that